c2corg_api/models/userdata.py

```diff
@@ -0,0 +1,58 @@
+import bcrypt
+
+from sqlalchemy import (
+    Column,
+    Integer,
+    String
+    )
+
+from c2corg_api.models import Base, users_schema
+
+
+class UserData(Base):
+    """
+    Class containing the users' private and authentication data.
+    """
+    __tablename__ = 'userdata'
+    __table_args__ = {"schema": users_schema}
+    __mapper_args__ = {
+        'polymorphic_identity': 'ud'
+    }
+
+    document_id = Column(Integer, primary_key=True)
```

Comment:
> Perhaps we should rename this field to "id" and add a FK towards the User (or Profile?) document_id

I would make a field `id` as PK and a field `document_id` as FK pointing to the profile (which is a document).